cypress/integration/editor/Extraction.feature

```diff
@@ -0,0 +1,27 @@
+Feature: Extraction Modal
+
+  I want to check that extraction modal is open when collection of items is larger than limit
+
+  Background:
+    Given I open a Data Explorer application in editor view
+    And Select "Extraction" analysis
+    And Select Event stream
+
+  @smoke-test
+  Scenario: Perform test for extracting a full collection of properties
+    When I click on "Preview events" button
+    Then Modal for "Large amount of properties" appears
+    And Close the modal
+
+  @smoke-test
+  Scenario: Perform test for selecting more properties than confirmExtractionLimit
```

Comment:
The scenario description should be written in `business` way without any implementation details like for eg. reference to variable names etc.
So this example scenario description could be like this: Shows extraction confirmation modal when count of properties exceed the limit